### PR TITLE
Update truss flag -build=false to -nobuild

### PR DESCRIPTION
**Pull request by hasAdamr**

### CODE REVIEW

File: truss/main.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func main() {
 
 	generator.GenerateMicroservice(genImportPath, workingDirectory, definitionFiles)
 
-	if *buildMicroservice {
+	if !*noBuild {
```
Comment (hasIan) [inline truss/main.go:77]: Heh, this looks really weird, but it makes the user interface more sensible. "If not no build."

File: truss/main.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ func init() {
 
 // Stages are documented in README.md
 func main() {
-	buildMicroservice := flag.Bool("build", true, "Set -build=false to not build microservice")
+	noBuild := flag.Bool("nobuild", false, "Set -nobuild to not build generated microservice")
```
Comment (hasIan) [inline truss/main.go]: "to not build" sounds a little funky; I'd probably say something like "to generate code without building," or something like that.

**Comment by hasIan** [comment]
Just one minor comment on the help text; address it however you see fit, and LGTM. 👍 :shipit:

File: truss/main.go (hunk 1 above)
Comment (hasAdamr) [inline truss/main.go:77]: Yeah, this is actually the reason I switched to -build=false. I did not like how the code looked.

**Pull request merged**